### ISSUE TITLE
RelVal input data placement and location must be done with another Rucio account

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#10254]
**Impact of the new feature**
MSTransferor

**Is your feature request related to a problem? Please describe.**
Yes, we would like to avoid blocking Release Validation workflows in a scenario where central production has exhausted all the `wmcore_transferor` quota at the usual target RelVal sites.

**Describe the solution you'd like**
We would like to separate central production from RelVal input data placement AND location in MSTransferor. This is the second out of 2 issues.

With this issue, we should implement the following:
* have a way to distinguish RelVal from other workflows in MSTransferor
* when running the input data location, only consider data that is locked by a relval account rule
* if input data is already available and locked by the relval account, then there is no need for any input data placement
* otherwise, create a rule against the RSEs where the workflow is meant to read data from (PSN x PNN). Of course, if the RSE does not have enough quota, we should make a record in the log and skip the input data placement until space is freed up.

In short, we need to implement all the input data location and input data placement for RelVal workflows using a (yet to be created) new Rucio account.

**Describe alternatives you've considered**
none

**Additional context**
It depends on this issue: https://github.com/dmwm/WMCore/issues/10253

**Comment by amaltaro** [comment]
Sigh, changes on the WorkQueue side are quite complicated! Passing Rucio accounts all the way from the top of the service (service configuration) down to the start policies is a lot of work.
It's not my preference, but an environment variable would make this development much simpler, and the code more maintainable as wel...

Given that we need to identify what is a RelVal workflow and what is not, it looks like this development:
https://github.com/dmwm/WMCore/issues/10282
is actually a prerequisite for this account separation feature. This should be straight forward, so I will try to get at least this in for testbed tomorrow.
